### ISSUE TITLE
Fix S2629 FP: Constant fields in interpolated string

### ISSUE THREAD

**Issue by Tim-Pohlmann** [SonarSource/sonar-dotnet#9247]
```cs
logger.Log(LogLevel.Warning, $"Constant: {FieldConstant}");  // Noncompliant FP
```

Originally reported by [community](https://community.sonarsource.com/t/s2629-false-positive-for-logging-string-interpolation-if-the-interpolated-in-are-placeholder-names/114661/5).
